### DISCUSSION TITLE
v1.0.3 ARM Trusted Firmware/U-Boot

### DISCUSSION THREAD

**Discussion by lectrical** [dangowrt/owrt-ubi-installer#161] [General]
Sorry if this is a dumb question but i just wondered (fomo), when I saw these release updated these things.

1: If i used v1.0.2 and updated to 23.05.0 from official source do I have old versions of these from v1.0.2?

2: Can I update my rt3200 using your release to get these updated firmwares versions? If yes, how?

3: Does this make any difference to me? (should i just ignore these version changes?)

4: edit: and thanks for the update!

**Comment by dangowrt** [comment]
> If i used v1.0.2 and updated to 23.05.0 from official source do I have old versions of these from v1.0.2?

The update of the OpenWrt image does not update the bootloader, hence you are still running the version you flashed from v1.0.2. But that's not a problem at all.

> Can I update my rt3200 using your release to get these updated firmwares versions? If yes, how?

You could connect the serial console and use the bootloader menu to feed update files for bl2 and bl31 via TFTP. Or you can re-run the installer (by flashing the unsigned installer image and overriding the image check ie. use the *force* option of sysupgrade).
As both come with a risk of bricking the device and there are no advantages in updating either TF-A or U-Boot on this device, I would not recommend anyone to do that.

> 3: Does this make any difference to me? (should i just ignore these version changes?)

It does not make a difference. Just ignore it.

There is no need to update the bootloader of a device which has previously been running an older version of the installer, from a user point of view it really doesn't make any difference at all and only presents an unnecessary risk to brick the device in the process of trying to update the TF-A and U-Boot.

**Reply by lectrical** [reply]
Thanks for the info. I'll just leave it as it is then.
